Resources/Prototypes/_RMC14/Entities/Objects/Medical/defib.yml

```
@@ -99,3 +103,56 @@
   parent: CMBaseDefibrillator
   id: CMDefibrillatorInfinite
   suffix: RMC14, Infinite
+
+- type: entity
+  parent: [CMBaseDefibrillator, RMCPowerCellSlotUnremovable]
+  id: RMCSynthResetKey
+  name: WeYa synthetic reset key
+  description: Result of collaboration between Cybersun and Weston-Yamada, this device can fix major glitches or programming errors of synthetic units, as well as being able to restart a synthetic that has suffered critical failure. It can only be used once before being reset.
```

Asked Whisper and we are leaving the cybersun replacement bit out for now until we figure out what we want to replace it with
```suggestion
  description: This device can fix major glitches or programming errors of synthetic units, as well as being able to restart a synthetic that has suffered critical failure. It can only be used once before being reset.
```